### ISSUE TITLE
Compile failure with base < 4.8

### ISSUE THREAD

**Issue by hvr** [tfausak/json-feed#1]
```
Configuring component lib from json-feed-0.0.1...
Preprocessing library json-feed-0.0.1...

library/JsonFeed.hs:27:8:
    Could not find module ‘Numeric.Natural’
    It is a member of the hidden package ‘nats-1.1.1’.
    Perhaps you need to add ‘nats’ to the build-depends in your .cabal file.
    Use -v to see a list of the files searched for.
```

I've already revised the affected release at https://hackage.haskell.org/package/json-feed-0.0.1/revisions/

**Comment by tfausak** [comment]
Blerg. Version 0.0.0 had the correct bounds. I changed them because the docs failed to build. https://hackage.haskell.org/package/json-feed-0.0.0/reports/3

**Comment by tfausak** [comment]
Fixed in version 0.0.2.
